py/server/deephaven/experimental/s3.py

```diff
@@ -76,6 +78,14 @@ def __init__(self,
                 anonymous access. Can't be combined with other credentials. By default, is False.
             endpoint_override (str): the endpoint to connect to. Callers connecting to AWS do not typically need to set
                 this; it is most useful when connecting to non-AWS, S3-compatible APIs.
+            part_size (int): the size of each part (in bytes) to upload when writing to S3, defaults to 10 MiB. The
+                minimum allowed part size is 5 MiB. Setting a higher value may increase throughput, but may also
+                increase memory usage. Note that the maximum number of parts allowed for a single file is 10,000.
+                Therefore, for 10 MiB part size, the maximum size of a single file that can be written is roughly
+                100k MiB (or about 98 GiB).
+            num_concurrent_parts (int): the maximum number of parts that can be uploaded concurrently when writing to S3
+             without blocking, defaults to 64. Setting a higher value may increase throughput, but may also increase
+             memory usage.
```

Comment:
What is a "part"?  This is not well conveyed in the docs.  When I Google "S3 Part", I get a bunch of race car parts -- so users can't self help.  A little elaboration / clarification will help reduce support costs.